stable/atlantis/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "v0.7.1"
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 3.5.3
+version: 3.5.4
```

```suggestion
version: 3.6.0
```
Since it's new functionality it should be a minor release